delfin/coordination.py

```diff
@@ -320,3 +322,35 @@ def _get_redis_backend_url():
             .format(backend_type=CONF.coordination.backend_type,
                     server=CONF.coordination.backend_server)
     return backend_url
+
+
+def on_node_join(event):
```

Comment:
where we need to register these callback?

Reply:
Code updated, we will register it on leader node